app/ceom/photos/templates/photos/leaderboard.html

```diff
@@ -0,0 +1,114 @@
+{% extends "photos/base.html" %}
+{% load i18n %}
+{% block content %}
+
+<h2 style="color: #0860be;" >Photos Leaderboard</h2>
+
+<h5 class="text-center"> Welcome to the <strong> CEOM Photos Leaderboard</strong>! </h5>
```

Comment:
Generally I don't use introduction messages like this

I would recommend just removing this line, given that it repeats the information in the line above it
I'll take a look at it whenever it's put into production and I might change my mind